app-misc/jigdo/jigdo-0.7.3.recipe

```diff
@@ -0,0 +1,60 @@
+SUMMARY="Ease the distribution of files"
+DESCRIPTION="Jigsaw Download, or short jigdo, is a tool designed to ease the distribution \
+of very large files over the internet, for example CD or DVD images. Its aim is \
+to make downloading the images as easy for users as a click on a direct \
+download link in a browser, while avoiding all the problems that server \
+administrators have with hosting such large files."
+HOMEPAGE="http://atterer.org/jigdo/"
+COPYRIGHT="2001-2006 Richard Atterer"
+LICENSE="GNU GPL v2"
+REVISION="1"
+SOURCE_URI="http://atterer.org/sites/atterer/files/2009-08/jigdo/jigdo-$portVersion.tar.bz2"
+CHECKSUM_SHA256="875c069abad67ce67d032a9479228acdb37c8162236c0e768369505f264827f0"
+PATCHES="jigdo-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 ?x86 x86_64"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	jigdo$secondaryArchSuffix = $portVersion
+	cmd:jigdo_file$secondaryArchSuffix = $portVersion
+	cmd:jigdo_lite$secondaryArchSuffix = $portVersion
+	cmd:jigdo_mirror$secondaryArchSuffix = $portVersion
```

Comment:
Since jigdo does not build on x86_gcc2 primary arch, it would be nice to have these changes:
```diff
 SECONDARY_ARCHITECTURES="x86"
 
+commandBinDir=$binDir
+commandSuffix=$secondaryArchSuffix
+if [ "$targetArchitecture" = x86_gcc2 ]
+then
+	commandBinDir=$prefix/bin
+	commandSuffix=
+fi
+
 PROVIDES="
 	jigdo$secondaryArchSuffix = $portVersion
-	cmd:jigdo_file$secondaryArchSuffix = $portVersion
+	cmd:jigdo_file$commandSuffix = $portVersion
-	cmd:jigdo_lite$secondaryArchSuffix = $portVersion
+	cmd:jigdo_lite$commandSuffix = $portVersion
-	cmd:jigdo_mirror$secondaryArchSuffix = $portVersion
+	cmd:jigdo_mirror$commandSuffix = $portVersion
 	"
```